### PR TITLE
Add Code of Conduct file

### PR DESCRIPTION
**Pull request by moria97**
Add back several manifest files that were removed when moving to main branch.

### CODE REVIEW

**Comment by qiwjin** [comment]
let's merge into dotliquid branch first.

**Review by qiwjin — APPROVED** [review]

**Review by BoyaWu10 — APPROVED** [review]

File: GeoPol.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <Component Include="List here folders to Include in a GeoPol Scan">    
+  <!-- . means the entire repo -->
+  <!-- Use back slash \ to indicate folder path e.g. src\Test -->
+    <IncludeFolder>.</IncludeFolder>
+  </Component>
+    <Component Exclude="List exceptions here to not be scanned, that have been included above">    
+    <!-- Use back slash \ to indicate folder path e.g. src\external\ -->
+    <ExcludeFolder>.gitignore</ExcludeFolder>
+    <ExcludeFolder>GeoPol.xml</ExcludeFolder>
+    <ExcludeFolder>test</ExcludeFolder>
```
Comment (JoeDRowan) [inline GeoPol.xml:21]: Sub folder to exclude 'test' no longer in repo so should be removed from this repo or replaced with correct path to test

**Review by JoeDRowan — COMMENTED** [review]
Sub folder in GeoPol.xml to exclude 'test' no longer in repo so should be removed from this repo or replaced with correct path to test

File: GeoPol.xml (hunk 1 above)
Comment (moria97) [reply GeoPol.xml:21]: Thanks. Removed it in pull request #387

**Pull request merged**